lib/themes/dosomething/paraneue_dosomething/js/onboarding.js

```diff
@@ -10,9 +10,13 @@ const ReactDom = require('react-dom');
 const Modal = require('dosomething-modal');
 
 function getLocalStorageKey(user, campaign) {
+  // @TODO: probably need a better way to get the run ID consistently.
+  const languageCode = (Object.keys(campaign.campaign_runs.current)[0]);
```

Comment:
Not a fan of this but seemed to work pretty well and more reliable than just relying on `campaign.campaign_runs.current.en.id`... thoughts?